apps/i18n/common/en_us.json

```diff
@@ -50,6 +50,9 @@
   "addStudentsSyncThirdParty": "Sync my list of students from an existing classroom section in a third party tool",
   "addStudentsToSectionInstructions": "Choose how you want to add your students:",
   "addStudentsToSectionInstructionsUpdated": "How do you want your students to sign in?",
+  "addStudentsToSectionPermissionHeader": "If students in your class are under 13 years old",
+  "addStudentsToSectionPermissionWarning": "We strongly recommend that you do not use personal logins. Using personal logins may require parental consent for security and privacy reasons.",
```

Comment:
One concern I have is it seems like we are trying to concatinate two parts of a sentence together, which doesn't necessarily work in every language (maybe the if/then structure is reversed.). I think the quick fix is to have the "heading" more like a label:
> Classes with students under 13 years old:

In the past, I have been reluctant to use colons in labels, but I might have been over sensitive. Alternatively you could put the Warning on a new line.

Reply:
Hmm. I removed the comma from the mock up for this reason. I was also questioning what to do here and settled on bold weight and color being distinguishable enough. Another thing would be to add some kind of stylized divider... em dash, etc, which could be a little more language agnostic. Do we know if that's a better best practice?